staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -260,7 +258,39 @@ func (o *CreateJobOptions) createJob() *batchv1.Job {
 	return job
 }
 
-func (o *CreateJobOptions) createJobFromCronJob(cronJob *batchv1beta1.CronJob) *batchv1.Job {
+func (o *CreateJobOptions) createJobFromCronJobV1Beta1(cronJob *batchv1beta1.CronJob) *batchv1.Job {
+	annotations := make(map[string]string)
+	annotations["cronjob.kubernetes.io/instantiate"] = "manual"
+	for k, v := range cronJob.Spec.JobTemplate.Annotations {
+		annotations[k] = v
+	}
+
+	job := &batchv1.Job{
+		// this is ok because we know exactly how we want to be serialized
+		TypeMeta: metav1.TypeMeta{APIVersion: batchv1.SchemeGroupVersion.String(), Kind: "Job"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        o.Name,
+			Annotations: annotations,
+			Labels:      cronJob.Spec.JobTemplate.Labels,
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					// TODO (soltysh): switch this to v1 when we switch controller in v1.22
```

Comment:
this is less about the controller and more about n-1 skew support in kubectl, right?

Reply:
is there an issue in the milestone tracking this TODO?

Reply:
> this is less about the controller and more about n-1 skew support in kubectl, right?

Correct, updated description. 

> is there an issue in the milestone tracking this TODO?

https://github.com/kubernetes/enhancements/issues/19, I'm collecting all the tasks in a single place.